students/synowiec_krzysztof/lesson_06_dicts_tuples_sets_args_kwargs/cubes.py

```diff
@@ -0,0 +1,29 @@
+def main():
+    n, m = [int(x) for x in input().split()]
+    alice_set = prepare_cubes(n)
+    bob_set = prepare_cubes(m)
+
+    intersection_set = sorted(alice_set.intersection(bob_set))
```

Comment:
it's totally fine, but just for a record. it's possible to use standard operators here, so this could look like:
`intersection_set = alice_set & bob_set`